docs/02-admin/04-running-mbin/05-cli.md

```diff
@@ -346,3 +346,13 @@ Usage:
 ```bash
 php bin/console mbin:user:create [-r|--remove] [--admin] [--moderator] [--] <username> <email> <password>
 ```
+
+### Update-Local-Domain
+
+This command will remove all remote posts from belonging to the local domain. This command is only relevant for instances
+created before v1.7.4 as the local domain was the fallback if no domain could be extracted from a post.
+
+Usage:
+```bash
+php bin/console mbin:update:local-domain [--]
```

Comment:
What is --?

Reply:
that's just something symfony creates automatically :D should I remove it?

Reply:
Uh. Yes please, maybe the command has an incorrect parameter config? Anyhow, I don't think it should have this optional --. Or at least remove it from the docs indeed, it's very confusing to me.